app/scripts/lib/multichain-api/caip-25.ts

```diff
@@ -141,3 +141,21 @@ export const isSupportedScopeString = (scopeString: string) => {
 
   return false;
 };
+
+export const flattenScope = (
```

Comment:
can we just add a jsdoc or comment indicating the formatting case (linking to CAIP-25) this is handling?

Reply:
added here https://github.com/MetaMask/metamask-extension/pull/25589/commits/936858ab0233cbe21f025cd11027b1fb422d1bc9